### DISCUSSION TITLE
cant unpack big file

### DISCUSSION THREAD

**Discussion by damiankomonski** [supabase/supabase#28505] [Feature Requests]
hello, there is problem with unpack big files in edge function, storage
1gb file, 2gb file

I want upload such file to storage, unpack it in edge function and then make some operations on files. I cant do this. Is it possible?

**Comment by damiankomonski** [comment]
No answer

**Comment by GaryAustin1** [comment]
https://stackoverflow.com/questions/75214131/reactjs-and-supabase-unzip-an-uploaded-file-and-store-in-db

This says no it can't be done from a year ago (from an employee) if unpack means unzip.

But you have a feature request here, and then you ask if it can be done.  So not sure if you are just suggesting something for Supabase to implement or asking if it can be done now.

**Comment by damiankomonski** [comment]
I am asking and if it isn’t possible it would be nice Supabase implement it, make something

**Reply by GaryAustin1** [reply]
K, it appears no from that link, I don't know of anything that has changed in edge functions since then.
And I'll leave this open as it is a feature request for them (SB).
